docs/guides/accessing-the-dom.md

```diff
@@ -8,9 +8,12 @@ order: 5
 
 Accessing the DOM directly in React is discouraged, because it pierces the DOM abstraction. Still there are some cases when InstUI is required to do so (for example for focus management or positioning). This happens with the [findDOMNode](#findDOMNode) utility function.
 
```

Comment:
We will need to compile a list for our users when they need to add this prop. The good news is that we have been spamming their console with warnings about this since InstUI 9